lib/helpers/combineURLs.js

```diff
@@ -8,5 +8,8 @@
  * @returns {string} The combined URL
  */
 module.exports = function combineURLs(baseURL, relativeURL) {
-  return baseURL.replace(/\/+$/, '') + '/' + relativeURL.replace(/^\/+/, '');
+  var end = relativeURL.replace(/^\/+/, '');
```

Comment:
I thought about this, and covered the case by using the full baseURL (it could include an ending slash).

```js
combineURLs('http://test.com/foo/', ''); // => 'http://test.com/foo/'
```

But yes I think it may be confusing without the support for your suggestion.